drools-parser/src/test/java/org/drools/parser/MiscDRLParserTest.java

```diff
@@ -2835,7 +2833,6 @@ public void parse_TypeDeclarationWithFields() throws Exception {
 
     }
 
-    @Disabled("Priority : High | Failed to parse or with parentheses in LHS")
     @Test
     public void parse_RuleWithLHSNesting() throws Exception {
```

Comment:
Hi @tkobayas : Does this (or other) test cover multi-level nesting ?
If not, could you please add one that cover that specific case (multiple level of nested parentheses) ?

Reply:
The test case is one level nest (renamed to `parenthesesOneLevelNestWithThreeSiblings`).

However, we already had `multipleLevelNestAndOrOrOrAnd()` (renamed).

I also added 2 test cases of tricky multiple level nesting : `multipleLevelNestWithThreeOrSiblings` and `existsMultipleLevelNestWithThreeOrSiblings`

Thanks!